### ISSUE TITLE
[ERROR] > < configure_fdu() > Error The "default" storage pool doesn't exist

### ISSUE THREAD

**Issue by zrnapoleon** [eclipse/fog05#148]
Meet error when try to deploy un lxd fdu:

Traceback (most recent call last):
  File "lifecycle_nonetwork.py", line 76, in <module>
    main(sys.argv[1], sys.argv[2])
  File "lifecycle_nonetwork.py", line 48, in main
    a.fdu.configure(instid)
  File "/usr/local/lib/python3.6/dist-packages/fog05-0.2.0-py3.6.egg/fog05/fimapi.py", line 618, in configure
  File "/usr/local/lib/python3.6/dist-packages/fog05-0.2.0-py3.6.egg/fog05/fimapi.py", line 503, in __wait_node_fdu_state_change
ValueError: Unable to change state to CONFIGURE for FDU Instance: 4e513bf2-b2f9-4c3f-90b2-8ed57370cb7e Errno: 2 Msg: The "default" storage pool doesn't exist

**Comment by gabrik** [comment]
This is because in the LXD plugin the storage pool that will be used is hard-coded.

Need to move this in the configuation file in the case of different configuration of the storage pool for LXD daemon

**Comment by zrnapoleon** [comment]
One more question:
My 2 node: 
192.168.12.196
192.168.12.202
![image](https://user-images.githubusercontent.com/53283508/64333580-6b00ac00-d009-11e9-9efb-83d33b513fe8.png)
I run the script on 196 and try to deploy fdu on 202, but this fdu can not be started
![image](https://user-images.githubusercontent.com/53283508/64333663-9c797780-d009-11e9-8c34-016ee9f070c0.png)

**Comment by gabrik** [comment]
Can you paste the output of the LXD plugin in the node that is not staring and also the output of 
```
lxc start cc0ec63c2-7c91-4263-9159-9a31293d8cc7
```

**Comment by zrnapoleon** [comment]
![image](https://user-images.githubusercontent.com/53283508/64334403-0cd4c880-d00b-11e9-8720-e9271cc3d87f.png)
mayb i need change the interface

**Comment by gabrik** [comment]
Yes, I guess that on the reconfiguration the name of the interface changed.

You can still create the same one, in this way you can use the same descriptor in both nodes.

```
lxc network create lxdbr0 
```

**Comment by zrnapoleon** [comment]
Yeah it is 
![image](https://user-images.githubusercontent.com/53283508/64334656-96849600-d00b-11e9-9065-efba142c5b17.png)
Thanks for your help!

**Comment by gabrik** [comment]
fixed by #149
